event_attribution_reporting_clicks.md

```diff
@@ -191,7 +191,7 @@ this API:
 to trigger attribution for all matching sources.
 
 The browser will treat redirects to a URL of the form:
-`https://<attributionreportto>/.well-known/attribution-reporting/trigger-attribution[?data=<data>&priority=<priority>]`
+`https://<attributionreportto>/.well-known/attribution-reporting/trigger-attribution[?trigger-data=<trigger-data>&priority=<priority>]`
```

Comment:
Can we update the name on row 198?

Reply:
Thanks for catching this! Fixed row 198.